### PR TITLE
correct the size of returned tensor in comments of ps_roi_pool.py and ps_roi_align.py

### PR DESCRIPTION
**Pull request by chengjuzhou**
This PR just corrects the size of returned tensor in comments and can be verified from 
https://github.com/pytorch/vision/blob/master/test/test_ops.py#L186
and
https://github.com/pytorch/vision/blob/master/test/test_ops.py#L390

### CODE REVIEW

File: torchvision/ops/ps_roi_align.py [hunk 1]
```diff
@@ -39,7 +39,7 @@ def ps_roi_align(
             ``ceil(roi_width / output_width)``, and likewise for height). Default: -1
 
     Returns:
-        Tensor[K, C, output_size[0], output_size[1]]: The pooled RoIs
+        Tensor[K, C/(output_size[0]*output_size[1]), output_size[0], output_size[1]]: The pooled RoIs
```
Comment (datumbox) [inline torchvision/ops/ps_roi_align.py:42]:
```suggestion
        Tensor[K, C / (output_size[0] * output_size[1]), output_size[0], output_size[1]]: The pooled RoIs
```

File: torchvision/ops/ps_roi_pool.py [hunk 2]
```diff
@@ -33,7 +33,7 @@ def ps_roi_pool(
             the box coordinates. Default: 1.0
 
     Returns:
-        Tensor[K, C, output_size[0], output_size[1]]: The pooled RoIs.
+        Tensor[K, C/(output_size[0]*output_size[1]), output_size[0], output_size[1]]: The pooled RoIs.
```
Comment (datumbox) [inline torchvision/ops/ps_roi_pool.py:36]:
```suggestion
        Tensor[K, C / (output_size[0] * output_size[1]), output_size[0], output_size[1]]: The pooled RoIs.
```

**Review by datumbox — COMMENTED** [review]
Thanks for the PR, the changes look good. 

Just added a bit of spacing to ensure that the Python lint won't complain. Let me know what you think.

File: torchvision/ops/ps_roi_pool.py (hunk 2 above)
Comment (chengjuzhou) [reply torchvision/ops/ps_roi_pool.py:36]: good, thanks

**Review by datumbox — APPROVED** [review]

**Pull request merged**